docs/architecture/state.md

```diff
@@ -18,70 +25,75 @@ The Miden Node(s) maintain three databases to describe the state:
 ![Architecture core concepts](../img/architecture/state/state.png){ width="80%" }
 </center>
 
+These databases are represented by authenticated data structures; for easily proving that items were added to or removed from a database, and so that a commitment to the database is very small.
```

Comment:
```suggestion
These databases are represented by authenticated data structures; for easily proving that items were added to or removed from a database, and so that a commitment to the database is very small.
```

Reply:
I stumbled upon this sentence, can you maybe rephrase it?

Reply:
Reads more simpler for me, in this way:

`These databases are represented by authenticated data structures that enable easy proof of items added to or removed from a database. This ensures that the commitment to the database remains very small.`